src/promptflow/promptflow/_sdk/entities/_trace.py

```diff
@@ -244,6 +244,11 @@ def _from_spans(spans: typing.List[Span]) -> "LineRun":
             else:
                 # eager flow/arbitrary script
                 main_line_run_data = _LineRunData._from_root_span(span)
+        # main line run span is absent, ignore this line run
+        # this may happen when the line is still executing, or terminated
```

Comment:
or one line is killed before the data is exported

Reply:
good catch, updated the comment